server/lorax_server/models/flash_causal_lm.py

```diff
@@ -186,13 +186,16 @@ def from_pb(
         if batch_tokenized_inputs is None:
             batch_inputs = []
             max_truncation = 0
-            for r in pb.requests:
-                inputs = tokenizers.get_inputs(r, tokenizer)
+            for request in pb.requests:
```

Comment:
updated from `r` to `request` since `pdb` treats `r` as return so you can't actually inspect this variable.